.github/actions/spell-check/excludes.txt

```diff
@@ -51,3 +51,4 @@ ignore$
 ^\.gitmodules$
 (?:^|/)WindowsSettings\.json$
 (?:^|/)timezones\.json$
+^\Q.pipelines/ESRPSigning_core.json\E$
```

Comment:
What does the `Q` do?

Reply:
`\Q`...`\E` is a way to not have to escape all the characters in between:
https://www.regular-expressions.info/characters.html#special